Gopkg.toml

```diff
@@ -289,14 +289,14 @@ required = [
 
 [[constraint]]
   name = "github.com/onsi/ginkgo"
-  revision = "ba8e856bb854d6771a72ddf6497a42dad3a0c971"
+  revision = "v1.7.0"
```

Comment:
I'm not sure if this is good TBH. 1.7 gingko added the JustAfterEach function and I'm not sure that works as we want/expect.

Reply:
It seems that everything is working as expected

Reply:
@eloycoto what would make you comfortable with using `v1.7.0`? Should failure cases in the Ginkgo tests be induced via adding something like `Expect(false).Should(BeTrue())` and see how `JustAfterEach` behaves in our CI?

Reply:
I think so, if something fails in the JustAfterEach the Fail will work(as normal) something that we don't want, due is not handle by ginkgo-ext, but I didn't test with 1.7 so I'm not sure about this. Just a concern here.